package com.tanya;

public class Triangle {

    public static double area(double base, double height) {
        return base * height / 2;
    }

    public static void main(String[] args) {
        System.out.println(area(5, 4));
    }
}
